### PR TITLE
sched hint

### PR DESCRIPTION
**Pull request by catlook**

### CODE REVIEW

**Comment by rackernexus** [comment]
Can one of the admins verify this patch?

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-a84a9f428a/how-to/). :bow:

File: content/cloud-servers/use-scheduler-hints-to-create-a-server.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+permalink: use-scheduler-hints-to-a-create-server/
```
Comment (stephamon) [inline content/cloud-servers/use-scheduler-hints-to-create-a-server.md]: Should probably fix the order of the wording here to be: `use-scheduler-hints-to-create-a-server/`

File: content/cloud-servers/use-scheduler-hints-to-create-a-server.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+You can create a server with scheduler hints by using curl calls or API browser
+plug-ins. Make sure that you have the server Universal Uniform Identifiers (UUIDs)
+for any servers that you want to build *close to* or *far away* from. The UUIDs
+can be found on the **Server Details** page in the **Servers** section of the
+[Cloud Control Panel](https:\\login.rackspace.com).
+
+In the following cURL examples, substitute the following variables with the
+corresponding values for your account:
+
+- region: The region of your new server.
```
Comment (stephamon) [inline content/cloud-servers/use-scheduler-hints-to-create-a-server.md]: Do you think we should bold these values, just so they stand out a bit from the explanation text?

File: content/cloud-servers/use-scheduler-hints-to-create-a-server.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+extension and provides an example of how to use it when building a cloud server.
+
+This process requires use of the API, so basic knowledge of how to generate
+authentication tokens and how to interact with the API by using curl is required.
+
+### The scheduler hint extension
+
+The scheduler hint extension enables you to specify the proximity of newly
+provisioned cloud servers to already existing cloud servers on your account.
+For more details on why this is important, please see our
```
Comment (cyrichardson) [inline content/cloud-servers/use-scheduler-hints-to-create-a-server.md]: important, see our

File: content/cloud-servers/use-scheduler-hints-to-create-a-server.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+[Cloud Server API Guide](https://developer.rackspace.com/docs/cloud-servers/v2/extensions/ext-sched-hint/).
+
+### Create a server with scheduler hints
+
+You can create a server with scheduler hints by using curl calls or API browser
+plug-ins. Make sure that you have the server Universal Uniform Identifiers (UUIDs)
+for any servers that you want to build *close to* or *far away* from. The UUIDs
```
Comment (cyrichardson) [inline content/cloud-servers/use-scheduler-hints-to-create-a-server.md]: You can find the UUIDS on the...

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-b0dcb778d5/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-b855c099c0/how-to/)

**Pull request merged**